src/presentation/kafka.py

```
@@ -15,24 +18,35 @@
-            account = core.receive_event.receive_event(msg.value["external_id"])
-        case "USER_CREATED":
-            account = core.account_create.create(msg.value["external_id"])
-        case "ORGANIZATION_CREATED":
-            account = core.account_create.create(msg.value["external_id"])
+try:
+    for msg in consumer:
+        console.info(msg)
+        core = Core()
+        try:
+            external_id = is_valid_uuid(msg.value["external_id"])
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)